page-templates/flexible.php

```diff
@@ -1,11 +1,17 @@
 <?php /* Template Name: Flexible */
 
-use \Timber\Timber;
-
-get_header();
+use Timber\Timber;
+use LeanNs\Assets;
 
 $data = Timber::get_context();
 $data['post'] = \LeanNs\Patterns::get_post();
+
+if ( 'images' === $data['post']->acf['hero']['hero_background']['type'] ) {
```

Comment:
Can we test if the `hero` is defined, since this can trigger an error on instances where this is not used or the fields does not exist, or we can say that is always there.